canonical/ubuntu.com
# Interactive forms

## Done

- Updated interactive form events to use the dataLayer instead of legacy GA event push

## QA

- Check out this feature branch
- Run the site using the command `./run serve` or `dotrun`
- View the site locally in your web browser at: http://0.0.0.0:8001/
    - Be sure to test on mobile, tablet and desktop screen sizes
- Click on a form that opens a multi-step modal and inspect the dataLayer object. You should see an "interactive-forms" event.
